06_puppy/willshen8/puppy.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+type Puppy struct {
+	ID    uint32
+	Breed string
+	Color string
+	Value string
+}
+
+func main() {
+	cutePuppy := Puppy{1, "Jack Russell", "White and Brown", "500"}
+	naughtyPuppy := Puppy{1234, "Fox Terrier", "Black", "1300"}
+	//
```

Comment:
Remove empty comment. Just use a blank line if you want to separate the sections.